dev_docs/tutorials/data/search.mdx

```diff
@@ -0,0 +1,490 @@
+---
+id: kibDevTutorialDataSearchAndSessions
+slug: /kibana-dev-docs/tutorials/data/search-and-sessions
+title: Kibana data.search Services
+summary: Kibana Search Services 
+date: 2021-02-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'search', 'sessions', 'search-sessions']
+---
+
+## Search Service
+
+### Low Level Search 
+
+Searching data stored in `Elasticsearch` can be done in various ways, for example using the `Elasticsearch` REST API or using an `Elasticsearch Client` for low level access.
+
+However, the recommended and easist way to search `Elasticsearch` is by using the low level search service. The service is exposed from the `data` plugin, and by using it, you don't only gain access to the data you stored, but also to additional capabilities, such as Custom Search Strategies, Asynchronous Search, Partial Results, Search Sessions and more.
+
+Here is a basic example for using the `data.search` service from a custom plugin:
+
+```ts
+import { CoreStart, Plugin } from 'kibana/public';
+import { DataPublicPluginStart, isCompleteResponse, isErrorResponse } from '???';
```

Comment:
```suggestion
import { DataPublicPluginStart, isCompleteResponse, isErrorResponse } from '../../src/plugins/data';
```

I understand that number of `../../` would be different for consumers, but I think it is better than `???`